src/sidebar/analytics.js

```diff
@@ -71,6 +71,8 @@ function analytics($analytics, $window, settings) {
       REPLY_CREATED: 'replyCreated',
       REPLY_UPDATED: 'replyUpdated',
       REPLY_DELETED: 'replyDeleted',
+      SHARED_DOCUMENT: 'sharedDocument',
+      SHARED_ANNOTATION: 'sharedAnnotation',
```

Comment:
The other events all have the form `OBJECT_VERB` rather than `VERB_OBJECT` so perhaps `DOCUMENT_SHARED` and `ANNOTATION_SHARED`?